src/Builder/Type/OutputWindow.php

```diff
@@ -62,7 +62,7 @@ public function __construct(
                 throw new InvalidArgumentException(sprintf('Expected $documents argument to be a list of 2 string or int. Got [%s, %s]', get_debug_type($documents[0]), get_debug_type($documents[1])));
             }
 
-            $window ??= new stdClass();
+            $window = new stdClass();
```

Comment:
Did static analysis complain about this? I assume `??=` was there because it was more flexible if a subsequent change resulted in this conditional no longer appearing first.

Reply:
Yes, static analysis.